### DISCUSSION TITLE
How do I use Pandoc to convert Latex to HTML such that the source code snippets inside Latex keep syntax highlighting in HTML?

### DISCUSSION THREAD

**Discussion by minoush82** [jgm/pandoc#10285] [Q&A] [answered]
I need to convert some Latex material into HTML using Pandoc. The issue I am facing is that the (C) source code snippets are not being highlighted in the HTML.

If I use the following as example:

```
\documentclass[a4paper]{report}
\usepackage{minted}
\begin{document}
    \begin{minted}[linenos, bgcolor=yellow, autogobble, tabsize=4]{c}
        void main()
        {
            printf("I need help!\n");
        }
    \end{minted}
\end{document}
```

Doing the conversion using `pandoc -N -f latex -t html example.tex -o example.html` yields an HTML file that looks as follows:
![example_html](https://github.com/user-attachments/assets/c96a20f8-7bdc-4420-8018-25c3443480ab)

Any ideas?

**Comment by massifrg** [comment, marked as answer]
The output is right, but you have no CSS attached to your HTML, because without the `-s`/`--standalone` option you only get a HTML fragment.

Try this instead:

```sh
pandoc -s -N -f latex -t html example.tex -o example.html
```

and you should see the output in colors.

**Reply by minoush82** [reply]
Wooha!! Many Thanks!
